test/integration/supported/client-execute-tests.js

```diff
@@ -600,8 +600,7 @@ describe("Client @SERVER_API", function () {
                 done,
             );
         }); */
-        // issues #79
-        /* vit("2.2", "should accept unset as a valid value", function (done) {
+        vit("2.2", "should accept unset as a valid value", function (done) {
```

Comment:
This test mentions `unset`, not `null`s. Isn't it contrary to the commit's description?

Reply:
The commit should enable both things